src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -52,11 +61,32 @@ public Climber() {
     m_rightPIDController.setI(ClimberConstants.kPID.i(), 0);
     m_rightPIDController.setD(ClimberConstants.kPID.d(), 0);
     m_rightPIDController.setFF(0);
+    m_state = State.PRE_FIRST_USE;
+  }
+
+  private boolean canMoveClimber(double position, double speed) {
+    if (
```

Comment:
This is a doozy of a boolean expression. I recommend using an outer switch statement to distinguish between states, and then see if it makes sense to break up the remaining expression further.